core/src/main/java/com/flipkart/varadhi/db/MetaStore.java

```diff
@@ -17,4 +19,9 @@ public interface MetaStore {
 
     VaradhiTopic getVaradhiTopic(String topicName);
 
+    void deleteVaradhiTopic(String topicName);
+
+    void deleteTopicResource(String projectName, String resourceName);
+
+    List<String> listVaradhiTopics(String projectName);
```

Comment:
do not implement this API name as of now. Needs some more thought on listing of Varadhi Topic objects as project name is embedded with in topic name and it would require some parsing.